test/views/pages/amends/OutcomeAmendedViewSpec.scala

```diff
@@ -0,0 +1,298 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.pages.amends
+
+import enums.ApplicationType
+import models.{AmendResultDisplayModel, PrintDisplayModel}
+import org.jsoup.Jsoup
+import play.api.i18n.Messages
+import testHelpers.ViewSpecHelpers.CommonViewSpecHelper
+import testHelpers.ViewSpecHelpers.amends.OutcomeAmendedViewSpecMessages
+import views.html.pages.amends.outcomeAmended
+
+class OutcomeAmendedViewSpec extends CommonViewSpecHelper with OutcomeAmendedViewSpecMessages {
+
+  val amendsActiveResultModelIP16 = AmendResultDisplayModel(
+    protectionType = ApplicationType.IP2016,
+    notificationId = 12,
+    protectedAmount = "£1,350,000.45",
+    details = Some(
+      PrintDisplayModel(
+        firstName = "Jim",
+        surname = "Davis",
+        nino = "nino",
+        protectionType = "IP2016",
+        status = "dormant",
+        psaCheckReference = "psaRef",
+        protectionReference = Messages("pla.protection.protectionReference"),
+        protectedAmount = Some("1,350,000.45"),
+        certificateDate = Some("14/07/2017"),
+        notificationId = 12
+      )
+    )
+  )
+
+  val amendsActiveResultModelIP14 = AmendResultDisplayModel(
+    protectionType = ApplicationType.IP2014,
+    notificationId = 5,
+    protectedAmount = "£1,350,000.11",
+    details = Some(
+      PrintDisplayModel(
+        firstName = "Jim",
+        surname = "Davis",
+        nino = "nino",
+        protectionType = "IP2014",
+        status = "dormant",
+        psaCheckReference = "psaRef",
+        protectionReference = Messages("pla.protection.protectionReference"),
+        protectedAmount = Some("1,350,000.11"),
+        certificateDate = Some("14/07/2017"),
+        notificationId = 5
+      )
+    )
+  )
+
+  lazy val outcomeAmendedView = fakeApplication().injector.instanceOf[outcomeAmended]
+  lazy val viewIP16           = outcomeAmendedView(amendsActiveResultModelIP16)
+  lazy val docIP16            = Jsoup.parse(viewIP16.body)
+
+  lazy val viewIP14 = outcomeAmendedView(amendsActiveResultModelIP14)
+  lazy val docIP14  = Jsoup.parse(viewIP14.body)
```

Comment:
There is a lot of conditions in `outcomeAmended` template, but only 2 test scenarios are present. I suggest to create a test scenario for each of the 14 notification IDs.